proposal: update Enhanced Scheduler Extension

eahydra opened the pull request:
Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### Ⅰ. Describe what this PR does

Update overview image and add comment to Hook

<!--
- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)
-->

### Ⅱ. Does this pull request fix one issue?

<!--If so, add "fixes #xxxx" below in the next line, for example, fixes #15. Otherwise, add "NONE" -->

### Ⅲ. Describe how to verify it

### Ⅳ. Special notes for reviews

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

hormes commented:
![image](https://user-images.githubusercontent.com/4395943/201290490-140cfc2c-e3cb-4563-8dee-85e1124d1467.png)

As we discussed, it actually belongs to a transformation inside the scheduling process, adjusted in place in one step?

eahydra commented:
> ![image](https://user-images.githubusercontent.com/4395943/201290490-140cfc2c-e3cb-4563-8dee-85e1124d1467.png)
> 
> As we discussed, it actually belongs to a transformation inside the scheduling process, adjusted in place in one step?

updated

hormes commented:
/lgtm
/approve
